README.md

```diff
@@ -58,6 +58,22 @@ See `THIRDPARTY-LICENSES` for third-party licenses' texts.
 
 This background of [this image](/public/images/og-image-sq.png) is courtesy NASA/JPL-Caltech.
 
+# Launching
+
+Download or clone repo. 
+- Use bundler `bundle install`
+- Run database migrations `rails db:migrate`
+- Create test data `rails db:seed`
+- Launch Rails server `rails s`
+- Visit to `localhost:3000`
+- Enter your email address in the input field (example@example.com).
+- Go to the console and copy the link. Follow this link. Example:
+  (`http://localhost:3000/sign_in/uR7S9GVX50Et6EMCMJ7MP2jPeCamf31USWKUSfEabEM`)
+- Visit to private page `http://localhost:3000/projects`
+- If need vizit public page run command `sudo nano /etc/hosts`
```

Comment:
To visit public pages on subdomains on your local machine you need to update your `/etc/hosts` file, adding the neede subdomain. For example: